docs/i18n.md

```diff
@@ -131,3 +131,16 @@ import { Trans } from 'react-i18next';
 
 // ...
 ```
+
+
+To activate translation on any branch add the following to the `config/env/local.js` inside the `module.exports = {}` code block:
+
+```
+featureFlags: {
+    reference: true, 
```

Comment:
Since `references` has nothing to do with i18n, let's not include it here as it can just confuse?

In the sample file there's a good example:

https://github.com/Trustroots/trustroots/blob/c30ce3b8a340537157f813462f77420a1e2a2a47/config/env/local.sample.js#L14-L17